### PR TITLE
Feature/error handling

### PR DESCRIPTION
**Pull request by krdurrett**
**What (if any) features are you implementing?**
Added  conditional rendering to show error messages from API calls.
Also added conditional rendering in case there is missing information in the selected movies object.

**What (if anything) did you refactor?**
Reformatted date to show regular US dating. Reformatted budget and revenue to have commas.

**Were there any issues that arose?**
n/a

**What outside research did you use (if any)?**
n/a

**Any other comments, questions, or concerns?**
n/a

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -65,10 +67,10 @@ class App extends Component {
     return (
       <main className='main'>
         <nav className='nav-bar'>
-          {/* remember to add onClick event listener to button */}
           <button onClick={this.returnHome} className='home-button'>🍿Rancid Tomatillos</button>
         </nav>
-        {this.displayMovieDetails()}
+        {this.state.error && <h2>Sorry, there was a problem with our network</h2>}
```
Comment (oacortez) [inline src/App.js:72]: Good use of conditional rendering. To display our network errors if any appears.

File: src/MovieDetail.js [hunk 2]
```diff
@@ -4,7 +4,11 @@ import './MovieDetail.css'
 const MovieDetail = ({ details }) => {
 
   let genres = details.genres.join(',');
-  
+  let rating = Math.round(details.average_rating * 100) / 100
```
Comment (oacortez) [inline src/MovieDetail.js:7]: Great use of Math round to update our average movie ratings

**Review by oacortez — APPROVED** [review]
Thank you, Kayla, for creating this error handling branch. I see You did some catch error handling and as well did some conditional rendering to not display if the movie detail has no values. Everything looks good to me! I will be merging this!

File: src/App.js [hunk 3]
```diff
@@ -21,14 +20,16 @@ class App extends Component {
         revenue: 0,
```
Comment (hannahhch) [inline src/App.js:20]: You don't _have_ to spell out every key here, but I'm sure it won't hurt. Just make sure you don't run into issues if the select movie has a piece of data missing. 

If you do really like to be specific about data types like this, you might want to look into Typescript!

File: src/App.js (hunk 1 above)
Comment (hannahhch) [reply src/App.js:72]: Agreed, this looks great!! As you start to see more opportunities for errors, you might even want to make an error component that takes in a dynamic message to display!

File: src/MovieDetail.js (hunk 2 above)
Comment (hannahhch) [reply src/MovieDetail.js:7]: Very nice touch here!

**Review by hannahhch — COMMENTED** [review]
Great job so far!

**Pull request merged**